app/kubemci/cmd/create.go

```diff
@@ -93,7 +93,7 @@ func NewCmdCreate(out, err io.Writer) *cobra.Command {
 	var options CreateOptions
 
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [lbname]",
```

Comment:
@perotinus @nikhiljindal Square brackets in CLIs are, by convention, used to denote optional arguments. It's not the case here. It should use angular brackets.

See git manpages or any other manpage for examples. Please fix.